docs/git-sync-relay.md

```diff
@@ -36,18 +36,28 @@ Create a `values.yaml` file with contents similar to the following:
 gitSyncRelay:
   enabled: True
   repo:
-    url: https://github.com/astronomer/airflow-example-dags
+    url: ssh+git://git@github.com/astronomer/2-4-example-dags.git # this can be https:// for public repositories
     branch: main
     depth: 1 # default to a shallow clone because it is faster, though it sacrifices git history
     wait: 60 # seconds between synchronizations with upstream git repo
     subPath: dags # if your dags dir is not the repo root, specify the path relative to the repo root
-    sshPrivateKeySecretName: theGitSshPrivateKeyName # The name of a secret that holds the private key.
+    sshPrivateKeySecretName: git-ssh-private-key # This is the secret we created earlier. This is not requred with https git remotes.
+
+airflow:
+  dags:
+    gitSync:
+      enabled: True
+      repo: git://airflow-git-sync-relay.${NAMESPACE}.svc.cluster.local./git
+      branch: main
+  dagDeployment:
+    repositoryUrl: https://github.com/astronomer/2-4-example-dags
+    type: git_sync
```

Comment:
This must be in the wrong place, that isn't in the OSS chart.

Reply:
That's bizarre. I did a deploy in dev and inspected the values and that's the structure I saw, so I configured it and it did work. I'm going to have to go back and look again and figure out wtf is going on because I don't see dagDeployment in either this chart or the OSS chart.

Reply:
Oh, my guess is that houston is storing state in the helm values. I'm doing another test now to see what the bare minimum instructions we need to include in doc to get it working are.

Reply:
I don't know what I was doing wrong before that was causing a crashloop, but when I try your example today it's working so yay.

Reply:
The best bugfix: time 🙃